### ISSUE TITLE
Question:  How to get details on ['Invalid index mnemonic']

### ISSUE THREAD

**Issue by dcslagel** [MandarJKulkarni/lascheck#4]
lascheck is reporting `['Invalid index mnemonic']` when I run the following steps:

```python
import lascheck
myfile = 'test.las'
print('-------------------------------------------')
las = lascheck.read(myfile)
print('-------------------------------------------')
print(myfile, las.check_conformity())
print('-------------------------------------------')
print(las.get_non_conformities())
```

Is there a function call to get more details on the error?  (Or future plans for adding that functionality?

Thank you,
DC

**Comment by MandarJKulkarni** [comment]
The term 'more details' needs to have a boundary. As of now, the function get_non_conformities gives hints about the non-conformities in the input file.

The readme of the package also lists the expectations from LAS standards.
e.g. "The only valid mnemonics for the index channel are DEPT, DEPTH, TIME, or INDEX."

The hints can be replaced with complete statements from the standards. Also, some hints can be improved to be more meaningful. 
e.g. "Missing a mandatory section" can be changed to indicate which exact section is missing.

This is in the plan.

**Comment by MandarJKulkarni** [comment]
I am marking this question as enhancement as we can add "more meaningful hints".

**Comment by dcslagel** [comment]
@MandarJKulkarni,

Your information above pointed me in the right direction.  The file I am looking at has this ~Curve section:
```
~Curve Information -----------------------------------------
TDEP .0.1 in : Tool Depth
DEPTH.m      : Depth Index
```

So the `TDEP` is the invalid index name and the `DEPTH` name (and column data) should be first.

The details that would have been helpful are that it was in the ~Curve Section and that 'TDEP' is invalid.  Optionally, even the line number of the non-conformity would be helpful. 

As you mentioned replacing the hint with the full text would be helpful.

Thank you!,
DC
